src/main/java/com/zocalo/shop/controller/ControllerExceptionHandler.java

```diff
@@ -23,4 +24,10 @@
         return new ResponseEntity<>(e.getMessage(), HttpStatus.BAD_REQUEST);
     }
 
+    @ExceptionHandler(UsernameNotFoundException.class)
+    @ResponseBody
+    public ResponseEntity<String> handleUsernameNotFoundException(UsernameNotFoundException e) {
+        return new ResponseEntity<>(e.getMessage(), HttpStatus.FORBIDDEN);
```

Comment:
User not found exception. Why forbidden code and not NotFound 404?

Reply:
It's a fair point.
Actually, I had my doubts about what to choose.
I have been guided by next ideas:
- I read HTTP specs https://tools.ietf.org/html/rfc7231#section-6.5.3, and decided that 403 could match situation, 
- Spring Security during unsuccessful authentication throws 404/401 codes
- I read, that's better to hide cause of unsuccessful authentication to prevent search for valid credentials by attackers.

But I re-read different sources, and I guess 404 could be fit better, because 403 specs telling about valid authentication as well (not our case, we have authentication failing), and if user make mistake/typo during login and then receive 403 he could be confused as well.

Thank you for comment! I will fix it